src/main/java/org/hawkular/client/android/activity/MetricDataActivity.java

```diff
@@ -80,7 +82,19 @@ private void setUpToolbar() {
     private void setUpMetricData() {
         showProgress();
 
-        BackendClient.getInstance().getMetricData(getTenant(), getMetric(), this, new MetricDataCallback());
+        BackendClient.of(this).getMetricData(
+            getTenant(), getMetric(), getMetricStartTime(), getMetricFinishTime(), new MetricDataCallback());
+    }
+
+    private Date getMetricStartTime() {
+        Calendar calendar = GregorianCalendar.getInstance();
+        calendar.add(Calendar.MINUTE, -10);
```

Comment:
In the UI the standard interval is currently 1h ( in RHQ we used to have 8h as default) - should the Android client do the same?